js/mainScript.js

```diff
@@ -27,17 +25,13 @@ function removeBook(e, newBookElement) {
   bookArray = bookArray.filter(checkBtnclicked);
   newBookElement.remove();
 }
-let entireJSON = '';
+
+const entireJSON = localStorage.getItem('bookKey');
 function syncStorage() {
-  entireJSON = localStorage.getItem('bookKey');
   if (entireJSON != null) {
-    localStorage.setItem(
-      'bookKey',
-      entireJSON.concat(JSON.stringify(bookArray)),
-    );
+    localStorage.setItem('bookKey', entireJSON.concat(JSON.stringify(bookArray)));
```

Comment:
- [ ] As noted by the previous reviewer, your data should be preserved in local storage after refreshing the page, meaning that the book shoold still be displayed after refreshing the page. Currently that is not the case https://www.loom.com/share/fb74208896764156ae10349e4c7f3717

Reply:
Hi GzimAsani. We have already solved it. Let me share my screen and show you if you want.

Reply:
> * [ ]  As noted by the previous reviewer, your data should be preserved in local storage after refreshing the page, meaning that the book shoold still be displayed after refreshing the page. Currently that is not the case https://www.loom.com/share/fb74208896764156ae10349e4c7f3717